sdks/python/sample/test-data/recordings/1bb8bed231ca5e50ed5cb037d5c4d33c6fdaa9357433c647c499c6903a69def3.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
This new file is part of the file upload, there was issue inside the genai python file upload, the headers are not passing through, have an internal fix for that.